source/Pacioli/Pacioli.Lib/Models/Journal.cs

```diff
@@ -1,13 +0,0 @@
-﻿using System.Linq;
-
-namespace Pacioli.Lib.Models
-{
-    public class Journal
```

Comment:
Journal was deleted as it was no longer covered by tests. JournalEntry now checks for balance.